### PR TITLE
feature/#2868-client-profile-page

### PR DESCRIPTION
**Pull request by rostikdoroshenko**
Created 'Дані користувача' page layout
Added base style
profile-page
profile-page2

### CODE REVIEW

File: src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.scss [hunk 1]
```diff
@@ -0,0 +1,98 @@
+.container {
```
Comment (JimHolman1199) [inline src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.scss:1]: let's not rewrite the bootstrap class, but use another one

Comment (rostikdoroshenko) [reply src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.scss]: fixed

**Review by YuriShevchuk — APPROVED** [review]

**Review by roksolanadutchak — APPROVED** [review]

File: src/app/main/component/ubs/components/ubs-submit-order/ubs-submit-order.component.html [hunk 2]
```diff
@@ -14,8 +14,10 @@
           <td class="firstRow"></td>
           <td class="firstRow right-column">
             <p class="total-content" *ngIf="certificatesSum">
-              <span>{{ 'order-details.bonuses' | translate }} </span><span>
-                <small>-{{ certificatesSum }} {{ 'order-details.currency' | translate }}</small></span>
+              <span>{{ 'order-details.bonuses' | translate }} </span
```
Comment (AlekOK) [inline src/app/main/component/ubs/components/ubs-submit-order/ubs-submit-order.component.html:17]: extra space

File: src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.html [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+        <h3>Додаткові дії</h3>
+      </div>
+
+      <div class="footer-btns">
+        <button class="btn btn-outline-success">Змінити пароль</button>
+        <button class="btn btn-outline-success">Видалити профіль</button>
+      </div>
+    </div>
+  </div>
+</div>
```
Comment (AlekOK) [inline src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.html:122]: In that file ubs-client-profile-page.component.html fix  grammar and remove extra space

File: src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.scss [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+    display: flex;
+
+    div {
+      margin-right: 30px;
+    }
+  }
+
+  label {
+    margin: 0;
+    color: grey;
```
Comment (AlekOK) [inline src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.scss:52]: grey ? use variable

File: src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.html (hunk 3 above)
Comment (rostikdoroshenko) [reply src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.html:104]: fixed

File: src/app/main/component/ubs/components/ubs-submit-order/ubs-submit-order.component.html (hunk 2 above)
Comment (rostikdoroshenko) [reply src/app/main/component/ubs/components/ubs-submit-order/ubs-submit-order.component.html:17]: fixed

File: src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.scss (hunk 4 above)
Comment (rostikdoroshenko) [reply src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.scss]: fixed

**Review by AlekOK — APPROVED** [review]

**Review by flypsoff — APPROVED** [review]

**Review by JimHolman1199 — APPROVED** [review]

File: src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.scss [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+  display: flex;
+  justify-content: center;
+  margin-bottom: 100px;
+  margin-left: auto;
+
+  button {
+    margin: 30px;
+  }
+
+  .header {
```
Comment (JimHolman1199) [inline src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.scss:13]: we already have header, let it be ubs_profile-header

File: src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.scss [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+
+  input.ng-invalid.ng-touched {
+    border: 1px solid var(--ubs-error-ligh-red);
+  }
+
+  h5 {
+    margin-left: 15px;
+  }
+
+  .google {
```
Comment (JimHolman1199) [inline src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.scss:67]: google is not the best selector

File: src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.scss [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+    }
+  }
+
+  .submit-btns {
+    width: 905px;
+    display: flex;
+    justify-content: flex-end;
+  }
+
+  .footer {
```
Comment (JimHolman1199) [inline src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.scss:86]: we already have footer, let it be ubs_profile-footer

File: src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.scss [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+      margin: 20px;
+    }
+
+    .footer-btns {
+      display: flex;
+      justify-content: flex-start;
+      box-shadow: 0 14px 28px rgba(0, 0, 0, 0.25);
+    }
+  }
+}
```
Comment (JimHolman1199) [inline src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.scss:99]: Screenshot 2021-06-21 at 17 57 37
please check responsive for a bigger screens

File: src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.scss (hunk 5 above)
Comment (rostikdoroshenko) [reply src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.scss]: done

File: src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.scss (hunk 6 above)
Comment (rostikdoroshenko) [reply src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.scss]: fixed

File: src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.scss (hunk 7 above)
Comment (rostikdoroshenko) [reply src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.scss]: done

File: src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.scss (hunk 8 above)
Comment (rostikdoroshenko) [reply src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.scss:99]: 2560x1875
fixed

**Review by JimHolman1199 — APPROVED** [review]

File: src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.html [hunk 9]
```diff
@@ -0,0 +1,104 @@
+<div class="profile-container">
+  <div class="row">
+    <div class="ubs_profile-header">
+      <div class="header-text">
+        <h3>Дані користувача</h3>
```
Comment (JimHolman1199) [inline src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.html:5]: move all text to it's .json files, so we can use translate here

**Review by JimHolman1199 — CHANGES_REQUESTED** [review]

File: src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.scss [hunk 10]
```diff
@@ -0,0 +1,99 @@
+.profile-container {
+  font-size: 0.9em;
+  width: 1200px;
+  display: flex;
+  justify-content: center;
```
Comment (Edw1x) [inline src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.scss:5]: These lines are repeating 4 times in this scss file. You can do @mixin for them.
But it's optional, as you want.

File: src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.html [hunk 11]
```diff
@@ -0,0 +1,104 @@
...
+        </div>
+        <div class="form-group col-sm-3">
+          <label for="district" class="form-label">Район</label>
+          <input *ngIf="editing" type="text" class="form-control" id="district" formControlName="district" />
+          <p *ngIf="!editing">{{ user.district }}</p>
+        </div>
+        <h5>Зв'язок із соцмережами</h5>
+        <div class="account">
+          <span>
+            <img src="http://pngimg.com/uploads/google/google_PNG19635.png" alt="google" />
```
Comment (JimHolman1199) [inline src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.html:84]: please use the existing google icon or upload a new one to asset folder

**Review by tkhristina — APPROVED** [review]

**Review by JimHolman1199 — APPROVED** [review]

**Pull request merged**